### ISSUE TITLE
Moving app windows to another desktop still does not work in Windows 11

### ISSUE THREAD

**Issue by everzwijn** [pmb6tz/windows-desktop-switcher#78]
Moving app windows to another desktop still does not work in Windows 11, instead the relevant desktop will be opened (no windows are moved).

**Comment by perettigiuliano** [comment]
with latest release it does not move to a desktop but switches to desktop

**Comment by strelov1** [comment]
I have the same problem

**Comment by d4em0n** [comment]
same problem

**Comment by ritchielrez** [comment]
What actually causes this problem, wish I knew autohotkey better, I could have fixed it then.

**Comment by ritchielrez** [comment]
**https://github.com/pmb6tz/windows-desktop-switcher/issues/72#issuecomment-972844002**, this post seems to fix the issue. Not sure if this breaks again after an update.

**Comment by Rednaxela5** [comment]
> **[#72 (comment)](https://github.com/pmb6tz/windows-desktop-switcher/issues/72#issuecomment-972844002)**, this post seems to fix the issue. Not sure if this breaks again after an update.

I just updated windows 11 and I noticed right away that moving app window to another desktop doesn't work. Thank you for linking this fix.

**Comment by ritchielrez** [comment]
> > **[#72 (comment)](https://github.com/pmb6tz/windows-desktop-switcher/issues/72#issuecomment-972844002)**, this post seems to fix the issue. Not sure if this breaks again after an update.
> 
> I just updated windows 11 and I noticed right away that moving app window to another desktop doesn't work. Thank you for linking this fix.

This solution might not feel smooth though. Like there's a delay between pressing a shortcut then switching the desktop.

**Comment by Rednaxela5** [comment]
> delay between pressing a shortcut then switching

Yup I've noticed that, especially when using different wallpaper per desktop. The taskbar tend to have a delay sometimes when switching rapidly.

**Comment by ritchielrez** [comment]
Hopefully this gets fixed, because this script is essential for my productivity.

**Comment by Rednaxela5** [comment]
> 

By the way, do you have any recommendation for "never combine taskbar tweak", here in windows 11?

**Comment by geekyBoogiepop** [comment]
Same problem here, it is so annoying! 😭

**Comment by felipecrp** [comment]
I also have the same problem in windows 11.

**Comment by httpsrichard** [comment]
Same problem :(
22H2

**Comment by mrroll** [comment]
> **[#72 (comment)](https://github.com/pmb6tz/windows-desktop-switcher/issues/72#issuecomment-972844002)**, this post seems to fix the issue. Not sure if this breaks again after an update.

For all those having issues with this., this dll works even on 22H2 for me.

More info here:

https://github.com/Ciantic/VirtualDesktopAccessor/issues/51#issue-1123702656
